### PR TITLE
solana: reorganize utility functions

### PR DESCRIPTION
**Pull request by aalu1418**
- [x] reorganize functions
- [x] remove `testing` from exported functions

### CODE REVIEW

**Review by jadepark-dev — APPROVED** [review]
Thank you for the effort on this thorough reorganization—it looks great to me!

File: chains/solana/contracts/tests/testutils/wrapped.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	"github.com/gagliardetto/solana-go"
+	"github.com/gagliardetto/solana-go/rpc"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/solana/gobindings/ccip_router"
+	"github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/ccip"
+	"github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/common"
+)
+
+// this files includes wrapped methods to be used in testing without additional error checks
```
Comment (agusaldasoro) [inline chains/solana/contracts/tests/testutils/wrapped.go:16]: I'm not sure if I follow why this file is named `wrapped` when it's all related to transactions and instructions, right?

**Review by agusaldasoro — APPROVED** [review]
Love this refactor!

File: chains/solana/contracts/tests/testutils/wrapped.go (hunk 1 above)
Comment (aalu1418) [reply chains/solana/contracts/tests/testutils/wrapped.go:16]: it's mostly just the actual implementation is elsewhere and these are test wrappers - i'm open to more suggested names if someone has a preference

**Pull request merged**